spec/fixtures/settings.yml

```diff
@@ -19,3 +19,7 @@ photo_sizes:
 root:
   yahoo.com: 2
   'google.com': 3
+
+feature:
+  enabled: true
+  group: x
```

Comment:
Instead of adding more sections to settings.yml, why don't we reuse what's already there?